docs/zephyr/quickref.rst

```diff
@@ -36,7 +36,10 @@ Use the :ref:`machine.Pin <machine.Pin>` class::
 
     from machine import Pin
 
-    pin = Pin(("GPIO_1", 21), Pin.IN)   # create input pin on GPIO1
+    gpiob = "gpiob"                     # GPIO port B devicetree node label
+    gpioc = "gpioc"                     # GPIO port C devicetree node label
```

Comment:
Is it worth defining variables like this?  The original example code had the strings written inline in the `Pin` constructor, and I think that's slightly better from an example point of view because it makes it easy to just cut-and-paste a single line and it should just work.  Eg:
```py
pin = Pin(("gpiob", 21), Pin.IN)
```
That's obvious at a glance just at that line how to specify a pin name.